R/preprocessing.R

```diff
@@ -646,7 +646,70 @@ Load10X_Spatial <- function (
   return(object)
 }
 
+#' Load a 10x Genomics Single Cell Experiment into a \code{Seurat} object
+#'
+#' @inheritParams Read10X
+#' @inheritParams SeuratObject::CreateSeuratObject
+#' @param data.dir Directory containing the H5 file specified by \code{filename}
+#' and the image data in a subdirectory called \code{spatial}
+#' @param filename Name of H5 file containing the feature barcode matrix
+#' @param filter.matrix Only keep spots that have been determined to be over
```

Comment:
(Preemptive-- this parameter doesn't seem to be implemented yet)
This sense of "filtering the matrix" is different than the more common one, which is removing barcodes from the counts matrix corresponding to empty and low-read GEMs. A user might have the misconception that they could set this option while providing the raw matrix to get the filtered matrix in the Seurat object.
To avoid confusion, how about calling this parameter something like `tissue.spots.only`?

Reply:
Oh, that makes sense. Thanks for clearing that up